### PR TITLE
Update processing_in_detail.md

### PR DESCRIPTION
**Pull request by dagewa**
Spellig

### CODE REVIEW

File: se-thaumatin/processing_in_detail.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ dials.find_spots imported.expt
 ```
 
 This is one of the two steps where every image in the data set is read
-and processed and hence can be moderately time consuming. This
+and processed and hence can be moderately time-consuming. This
```
Comment (graeme-winter) [inline se-thaumatin/processing_in_detail.md:189]: The man likes his hyphens 😉

File: se-thaumatin/processing_in_detail.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ than as a zip file, as some systems struggle with zip files > 4GB.
 
 ## Files
 
-DIALS creates two principle file types:
+DIALS creates two principal file types:
```
Comment (graeme-winter) [inline se-thaumatin/processing_in_detail.md:55]: 👍

File: se-thaumatin/processing_in_detail.md [hunk 3]
```diff
@@ -364,7 +364,7 @@ should not be a cause for concern.
 Once you have indexed the data you may optionally attempt to infer the
 correct Bravais lattice and assign this to constrain the unit cell in
 subsequent processing. If, for example, the unit cell from indexing
-has all three angles close to 90 degrees and two unit cell lengths
+has all three angles close to 90° and two unit cell lengths
```
Comment (graeme-winter) [inline se-thaumatin/processing_in_detail.md:367]: I was working in an emacs buffer at the time so couldn't be bothered to copy / paste in

File: se-thaumatin/processing_in_detail.md [hunk 4]
```diff
@@ -498,7 +498,7 @@ than e.g. a laptop.
 
 If you know in advance that the data do not diffract to anything close
 to the edges of the detector you can assign a resolution limit at this
-stage by adding `prediction.d_min=1.8` (say) to define a 1.8A
+stage by adding `prediction.d_min=1.8` (say) to define a 1.8 Å
```
Comment (graeme-winter) [inline se-thaumatin/processing_in_detail.md:501]: Likewise 👍

**Review by graeme-winter — APPROVED** [review]
I appreciate the contributions - are you done or is this still work in progress?

**Comment by dagewa** [comment]
I think done now :-)

I promise I did read _the content_ as well and found nothing wrong, so this is really just tidying using the strict settings in my English compiler

**Pull request merged**